src/test/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCallerSuite.scala

```diff
@@ -21,188 +21,138 @@ class GermlineAssemblyCallerSuite extends FunSuite with Matchers with BeforeAndA
   var reference: ReferenceBroadcast = _
   var readSet: ReadSet = _
 
-  before {
+  override def beforeAll() {
```

Comment:
`before` and `after` (defined in [`BeforeAndAfter`](https://github.com/scalatest/scalatest/blob/release-2.2.1-for-scala-2.11-and-2.10/src/main/scala/org/scalatest/BeforeAndAfter.scala#L21)) run for each "test".

`beforeAll` and `afterAll` (cf. [`BeforeAndAfterAll`](https://github.com/scalatest/scalatest/blob/release-2.2.1-for-scala-2.11-and-2.10/src/main/scala/org/scalatest/BeforeAndAfterAll.scala#L19)) run for each *suite*.

The definitions are different for whatever reason so I can't just say

```
beforeAll {
  …
}
```

but have to use the `override def beforeAll() { … }` you see here.